LearningJournal/LearningJournal/forms.py

```diff
@@ -0,0 +1,17 @@
+"""Define form classes."""
+
+from wtforms import Form, StringField, validators
+
+
+class EntryForm(Form):
+    """Create edit/create Entry form."""
+
+    title = StringField('Title', [validators.Length(min=4, max=25)])
+    text = StringField('Text', [validators.Length(min=6)])
+
+
+class LoginForm(Form):
+    """Create log in form."""
+
+    username = StringField("Username", [validators.Length(min=4, max=25)])
+    password = StringField("Password", [validators.Length(min=6)])
```

Comment:
Try using `PasswordField` here instead of `StringField`, so that a user's password isn't visible as plaintext when they type into the login form.